HW_1/TASK4.py

```diff
@@ -0,0 +1,17 @@
+
+"""
+Петя, Катя и Сережа делают из бумаги журавликов.
+Вместе они сделали S журавликов. Сколько журавликов сделал каждый ребенок,
+если известно , что Петя и Сережа сделали одинаковое количество журавликов,
+а Катя сделала в 2 раза больше журавликов, чем Петя и Сережа вместе?
+
+"""
+
+n = int(input("Введите количество журавликов"))
+
+if n % 2 == 0 and n % 3 == 0:  # Чтобы получить точный результат число должно делиться на 3 без остатка
+    a = int(n / 6)   # Если Катя сделала в два раза больше журавликов, чем мальчики вместе, то она сделала 2/3 журавликов
```

Comment:
Функции использовать совершенно необязательно. Оператор целочисленного отделение работает не хуже.
```suggestion
    a = n // 6   # Если Катя сделала в два раза больше журавликов, чем мальчики вместе, то она сделала 2/3 журавликов
```